azure-pipelines.yml

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+trigger:
+  branches:
+    include:
+    - '*' 
+  paths:
+    exclude:
+    - docs/*
+    - '*.md'
+
+resources:
+  containers:
+  # Container with Maven 3.6.1 to have the same environment everywhere.
+  - container: flink-build-container
+    image: rmetzger/flink-ci:2
+
+jobs:
+  # use 'Default' pool
+- job: runOnDefaultAgentPool
+  pool:
+    name: Default
```

Comment:
do I understand it correctly that this here runs on free machines, while the other job runs on sponsored ones?

Reply:
The "Default" pool are the sponsored machines. The name is a bit unfortunate, but we can change it